.github/workflows/php.yml

```diff
@@ -0,0 +1,56 @@
+name: PHP Composer
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+
+    # ubuntu contains php by default (2022)
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Clone matomo-org/matomo
+      uses: actions/checkout@v2
+      with:
+        repository: matomo-org/matomo
+        submodules: true
+        #ref: 4.7.x
+        
+
+    - name: Validate composer.json and composer.lock
+      run: composer validate --strict
```

Comment:
This step validates the composer settings of Matomo, not of our plugin. And apparently they're using an outdated license description, causing this step to fail. You can remove it completely, IMHO.